thumbor_request_modifier/__init__.py

```diff
@@ -0,0 +1,2 @@
+__author__      = "Maksim Barouski"
```

Comment:
I think this should be `DFFRNT Lab` as well

Reply:
We can also leave it off in this case; but I wouldn't mind keeping it around.